src/chrome/content/rules/CDG_Commerce.com.xml

```diff
@@ -1,4 +1,11 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://cdgcommerce.com/ => https://cdgcommerce.com/: (28, 'Operation timed out after 15001 milliseconds with 0 bytes received')
+Fetch error: http://myapp.cdgcommerce.com/ => https://myapp.cdgcommerce.com/: (28, 'Operation timed out after 15001 milliseconds with 0 bytes received')
+Fetch error: http://secure.cdgcommerce.com/ => https://secure.cdgcommerce.com/: (28, 'Operation timed out after 15001 milliseconds with 0 bytes received')
+Fetch error: http://www.cdgcommerce.com/ => https://www.cdgcommerce.com/: (28, 'Operation timed out after 15001 milliseconds with 0 bytes received')
+
```

Comment:
This failure is mysterious. I can reliably reproduce it with the checker but not in the browser or with command line curl. Reverting it for now, will investigate. https://github.com/hiviah/https-everywhere-checker/issues/10